### DISCUSSION TITLE
Observer location outside our Milky Way Galaxy

### DISCUSSION THREAD

**Discussion by HalleySTW** [Stellarium/stellarium#3223] [Q&A] [answered]
I should like to use the Planetarium functionalities of Stellarium to show the position of our Solar system in the Milky Way. And I should like to fly from a position far in deep space to our Earth.
There is an option in Stellarium to change the ObserverLocation. Stellarium can move the ObserverLocation to all the planents and moons of our solar system. But what I need is an option to move the ObserverLocation to 'Miky Way Observer' or 'Andromeda Observer'.
Is such an option available in Stellarium and if not, is it possible to 'trick' it (see discussion https://github.com/Stellarium/stellarium/discussions/2108#discussion-3748546)

**Comment by gzotti** [comment, marked as answer]
No, there is no way in Stellarium to place yourself outside of the Solar system. No interstellar spaceflight, no constellations as seen from Vega. This is simply not planned unless you do it. Use SpaceEngine or OpenSpace for that.

**Reply by HalleySTW** [reply]
Ok, thanks. Thats clear. I wanted to be sure.
